packages/patterns/src/keys/merge-set-operators.js

```diff
@@ -25,7 +25,7 @@ const { quote: q, Fail } = assert;
  *  to `fullOrder`. However, it optimizes for the case where these contiguous
  * runs that need to be resorted are either absent or small.
  *
- * @template T
+ * @template {import('../types').Passable} T
```

Comment:
I don't think the functions in this file need to be constrained to collections of Passables.

Reply:
They compare T using `RankCompare` whose documentation says,
https://github.com/endojs/endo/blob/40f6cf8794f9c67b3f5a65f01b7037ab960149bc/packages/marshal/src/types.js#L163-L166

Reply:
Ah, I forgot about that.